sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/AggregationQuerySuite.scala

```diff
@@ -930,6 +930,69 @@ abstract class AggregationQuerySuite extends QueryTest with SQLTestUtils with Te
         Row(11) :: Nil)
     }
   }
+
+  test("SPARK-14495: two distinct aggregation with having clause of one distinct aggregation") {
```

Comment:
these 4 tests look a little verbose to me. The key to trigger this bug is to put distinct aggregate function in having, right?